src/objective-c/tests/ThirdPartyTests/Libuv/LibuvGetAddrInfoTests.m

```diff
@@ -1,141 +1,35 @@
-/*
```

Comment:
Did you intend to remove the copyright notice?

Reply:
no, thank you for catching it